specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-11-11-preview",
+    "title": "Azure Migrate",
+    "description": "Move your workloads to Azure.",
+    "contact": {
+      "name": "Microsoft",
+      "email": "azuremigrate@microsoft.com"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "parameters": {
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "description": "Azure Subscription Id in which project was created.",
+      "type": "string",
+      "in": "path",
+      "required": true
+    },
+    "resourceGroupName": {
+      "name": "resourceGroupName",
+      "description": "Name of the Azure Resource Group that project is part of.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "projectName": {
+      "name": "projectName",
+      "description": "Name of the Azure Migrate project.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "machineName": {
+      "name": "machineName",
+      "description": "Unique name of a machine in private datacenter.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "assessedMachineName": {
+      "name": "assessedMachineName",
+      "description": "Unique name of an assessed machine evaluated as part of an assessment.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "groupName": {
+      "name": "groupName",
+      "description": "Unique name of a group within a project.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "assessmentName": {
+      "name": "assessmentName",
+      "description": "Unique name of an assessment within a project.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "api-version": {
+      "name": "api-version",
+      "description": "Standard request header. Used by service to identify API version used by client.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "enum": [
+        "2017-11-11-preview"
+      ]
+    },
+    "contentType": {
```

Comment:
>contentType [](start = 5, length = 11)

Is there a case in which this would be different than `application/json`?

Reply:
It would always be application/json.
Removing contentType parameter.

Reply:
Fixed